### ISSUE TITLE
Create result analysis notebook for VAE experiment

### ISSUE THREAD

**Issue by ltiao** [ltiao/anmoku#3]
All the plot generation code we got rid of from the VAE experiment should be turned into a Jupyter Notebook that loads the results Dataframe from the HDF5.

**Comment by ltiao** [comment]
By friday, we specifically need

- Plot discriminator binary crossentropy loss on twin y log scale with nelbo loss
- Plot for more than one MC sample for the ELBO
- Plot for full validation set, alongside training batches
- Plots across different runs

**Comment by ltiao** [comment]
Later on, we'd also like

- Plot for more than one MC sample for the discriminator binary crossentropy loss
- Plots for different adversarial training schedules
